### ISSUE TITLE
Bug in faas-cli cloud seal --namespace parameter

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas-cli#708]
## Expected Behaviour

The namespace has a default parameter, but it's not working:

```
	cloudSealCmd.Flags().StringVarP(&namespace, "namespace", "n", "openfaas-fn", "Secret name")

```

This should look like:

```
Sealing secret: alexellis-ofc-k3d in namespace: openfaas-fn

secrets.yml written.
```

## Current Behaviour

It looks like this though, with no namespace at all

```
Sealing secret: alexellis-ofc-k3d in namespace: 

secrets.yml written.
```

## Possible Solution

Investigate what caused the regression and fix it.

## Steps to Reproduce (for bugs)

1. `faas-cli cloud seal --name alexellis-ofc-k3d --literal secrets=kind -c ./cert.pem`
2. Look at `secrets.yaml`

## Context

This will be affecting OFC users that need secrets.

## Your Environment

* Operating System and version (e.g. Linux, Windows, MacOS):

MacOS

CLI:
 commit:  5ff1504047756f9ff2e0a10ae64803ab63df470a
 version: 0.9.3

* Link to your project or a code example to reproduce issue:

cert.pem:

```
-----BEGIN CERTIFICATE-----
MIIErjCCApagAwIBAgIRAL9h26Jjj/sm5UEYT2DihtcwDQYJKoZIhvcNAQELBQAw
ADAeFw0xOTEwMTcxNzI5MDdaFw0yOTEwMTQxNzI5MDdaMAAwggIiMA0GCSqGSIb3
DQEBAQUAA4ICDwAwggIKAoICAQDHUzh6FHmD5zB+ySzgMitFBNEwBMnHcTn/SOQZ
CA/oQ3ApgYKygGzZMM3hemo6/u7lit9B+01bJgajTnvJj40SQkCNyfDp3Xal4atR
R74hkhFMuR92JKepQ1FA3kF7lh+gjNoOdKrUurWAQHNIU2PPGgJ5KOFh5uak/tUm
v+EDJCPtrYqwZr/RL3su3Dr06N/13/tO1cCUum9jVBnuX0zoE4c53H3+HGwOphyA
03TBcfulOiGDiLT1CX7iGZdWFSkZLuFH3BIGyOjrfKDFWgFNBb2z8jfJUjzCqsYc
W+SLq4FuM8JJt/3ZPut5XbIgc6BIo6vh7IeMJ7ALMbDBgfmMkpZsVX4rVbw4DsTh
ZiFF2fETxqr3tDAn+9Pfs+UhhHEL/tCxfyVIwBc7JOiOd0OjiV08ycP//Be6/om5
PWEDhRveFRVooTH1j/dMgSvvyj+8iQg79W0VsifZFNrrAqwEbKLwcg79ASRd1JCz
+F5W43MMmCFoE+Po8r4Zs7AKf0sa2PoCNslU0rLyUygTUEwUvBWxtIxMlTKMCHoo
5RQyVEDzgzHhIdRy1efSZWsrnEmbCJ5+Ww7hUUo3FX+DcWXhuw11VMGhQU7npHtJ
qL9fzR7X4+C4XN1ULAV6skv4csOHKYdCk4erW4DQh7QqUbf+LOTchBcRMW/ELjg2
YrZ+qwIDAQABoyMwITAOBgNVHQ8BAf8EBAMCAAEwDwYDVR0TAQH/BAUwAwEB/zAN
BgkqhkiG9w0BAQsFAAOCAgEAUvCjKGXFvLXXiIBQk3dlu5RBrqqL6MCFF6oSjOg3
M+843CLG4jJmBO55SpLfXJE1XUGEhbELnuPaWMKPkueGT8mVmhoxD74jdv8Jmkvv
wb0ZuOanQdKPNAcJWLb735V1XXFTPudhOzedgFOzYBJCXJbaOCEgcC7iWA2QwCau
r/E1KC80Oi8PorXS3NnOIvLXiyS0nLJgnCny8OpaBXny2nAgWvKIzf0G1bNKwzeB
LeaoPoKtXxQ4FCisKIpy8V01EwFU8k5iU3VPzzFihEtRGKE5Rl5sDTZOkXau8H/L
7x0UjQHwYeOK9Wpq5J5WwbgZNFOEvILwBgVSBk6gdcVG7azovSo07pCCDh5nbeGW
7Lx+LgamscM0/9q0wxRb7HT2s9WUEZpuPCDMXt0QIMWcW0ZLEIm/JDeJbX7mfbjf
TFOOeos2G5CVKTBxliMdxmD6WvbZbbmz8M9bR+kIJxoozqbXohwDF5e4AlFhKWNI
+liy/8slAqofk00/zgAgXZHoZ/PwDAiB7WHx304uJsUQQX+OhV/zzc0fiKYl7om/
0mK1z0QJx47oERZsERHRmRdXPU3TFxC8KQrYGvcEB9q4VuqfSgbd6QE/IeanXcpS
8HzgBGyPgdNl2KtJ3loMVzggfW99LrT5xoO93yKl5WwWA9ny0olB3tQG0rhyrpwJ
cao=
-----END CERTIFICATE-----
```

**Comment by alexellis** [comment]
@viveksyngh PTAL?

**Comment by alexellis** [comment]
Fixed via #709
